serving/src/main/java/feast/serving/configuration/ServingServiceConfig.java

```diff
@@ -117,7 +117,8 @@ public ServingService servingService(
                 .setStorage(storage)
                 .build();
 
-        servingService = new HistoricalServingService(bqRetriever, specService, jobService);
+        servingService =
+            new HistoricalServingService(bqRetriever, specService, jobService, jobStagingLocation);
```

Comment:
Trust me I really want to greenlight this PR, but I don't think adding `stagingLocation` directly to the `HistoricalServingService` is correct here. That state is already stored within the retriever, which means it is also available to the HistoricalServingService if we expose it.

Reply:
If you are happy with me making the change then I can quickly do it and you can see if you agree with the structure?